### PR TITLE
Added Spock dependency and configure unit tests execution. Resolve…

### PR DESCRIPTION
**Pull request by mariuszkumor**
…d #5

### CODE REVIEW

**Comment by mariuszkumor** [comment]
Hmm... I'm confused, why Sonar didn't run unit tests.

File: pom.xml [hunk 1]
```diff
@@ -5,7 +5,47 @@
...
+            <scope>test</scope>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <!-- JUnit 5 requires Surefire version 2.22.0 or higher -->
+            <plugin>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <version>${maven.surefire.plugin.version}</version>
```
Comment (stlhrt) [inline pom.xml]: can't we just target maven 2.5 and use maven wrapper ?

File: src/test/java/org/powerflows/dmn/model/SampleTests.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+package org.powerflows.dmn.model;
+
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class SampleTests {
```
Comment (stlhrt) [inline src/test/java/org/powerflows/dmn/model/SampleTests.java]: tbh... Spock still rocks ;)

**Comment by mariuszkumor** [comment]
Finally, I decided to go with Spock.

**Pull request merged**